src/stardust/cli.py

```
@@ -1,26 +1,37 @@
-#!/usr/bin/env python3
 import argparse
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)